### DISCUSSION TITLE
Adjustable visibility of timer while typing

### DISCUSSION THREAD

**Discussion by whykay-01** [monkeytypegame/monkeytype#3921] [Ideas] [answered]
Hi everyone! Monkeytype is a great platform to work on our typing speed, however I believe that it would be even more powerful if users could adjust the visibility of the timer. For me personally, it is kind of annoying, or rather distracting that the timer is there when I am typing. What do you think? Would it be fair to provide an option for users to choose from (whether the timer should or should not be displayed while typing)?

**Comment by Miodec** [comment, marked as answer]
Settings > Timer / progress > off
